cartridge/failover.lua

```diff
@@ -192,6 +200,49 @@ local function apply_config(mod)
     )
 end
 
+local function reconfigure_all()
+    local confapplier = require('cartridge.confapplier')
+    local all_roles = require('cartridge.roles').get_all_roles()
+
+    -- WARNING: implicit yield
+    -- The event may arrive while two-phase commit is in progress.
+    -- We should wait for the appropriate state.
+    local state = confapplier.wish_state('RolesConfigured', math.huge)
+    fiber.testcancel()
+
+    if state ~= 'RolesConfigured' then
+        log.info('Skipping failover step - state is %s', state)
+        return
+    end
+
+    fiber.self().storage.is_busy = true
```

Comment:
May be "configuration_is_in_progress" is better

Reply:
I would rather not. It doesn't add much new information.